js/src/forum/components/PostStreamScrubber.js

```diff
@@ -31,8 +34,17 @@ export default class PostStreamScrubber extends Component {
       count: <span className="Scrubber-count">{formatNumber(count)}</span>,
     });
 
+    const index = this.stream.index;
+    const previousIndex = this.stream.previousIndex;
+
+    // We want to make sure the back button isn't crammed in.
+    // If the previous post index is less than 5% from the last/first post,
+    // or if the previous post index is less than 25% from the current post, we will
```

Comment:
I think that 25% threshold might be a bit much; it's confusing when the button shows up some times but not others.